package exceptionhandling;

public class TryFinally {
	
	public static void main(String[] args) {
		
		try {
			
			int a = 10/2;
			
		}
		
		finally
		{
			System.out.println("Finally block");
			
		}
	}

}
